crates/swc_html_minifier/tests/fixture/element/script-options-2/output.min.html

```diff
@@ -0,0 +1,5 @@
+<!doctype html><html lang=en><title>Document</title><div>Script:</div>
+<script>/* Should mangle top level stuff */ var o=function(){let o="bar";o&&(o+="baz"),console.log(o)}</script>
```

Comment:
Here another bug with printing comments - space between `/* Should mangle top level stuff */` and `var o`, we can should avoid it, because it increase size of bundle (I think this bug happens everywhere with comments), I will fix it in another PR